src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Apprenticeship.cs

```diff
@@ -144,5 +151,77 @@ public DraftApprenticeship CreateCopyForChangeOfParty(ChangeOfPartyRequest chang
 
             return result;
         }
+
+        private bool IsWaitingToStart(ICurrentDateTime currentDateTime)
```

Comment:
I believe there's an extension method for this.

Reply:
There is but it's in the V1 Code, I've ported it to V2, then made use of an extension method instead.